### ISSUE TITLE
Simplify start procedure for JSkat

### ISSUE THREAD

**Issue by b0n541** [b0n541/jskat-multimodule#17]
New users sometimes don't know how to start this JAR-file thing.
- explain it in detail on the homepage
- find a way to create a selfcontained ZIP archive with JRE and EXE file for Windows users

**Comment by mlrdev** [comment]
There's at least one handy way to do it manually :-p
http://launch4j.sourceforge.net/
http://sourceforge.net/projects/launch4j/#screenshots

But I assume this improvement aims at an automated workflow, right?

**Comment by b0n541** [comment]
You are right. I think a direkt solution from the JDK is a better option. JavaFX will help here too:

http://docs.oracle.com/javafx/2/deployment/self-contained-packaging.htm

**Comment by b0n541** [comment]
Meanwhile JPackage is the default option for creating self-contained, OS specific installers:

https://github.com/heshanthenura/JavaPackageDemo

It also supported by GitHub actions:

https://dev.to/sualeh/how-to-create-platform-specific-installers-for-your-java-applications-from-github-actions-2c15

**Comment by b0n541** [comment]
Is superseded by #196.
